tinygrad/opsgpu.py

```diff
@@ -131,7 +131,7 @@ def reduce_op(ctx, code, code2, inp, axis=None):
     osize = [1]*len(inp.shape)
   else:
     osize = np.array(inp.shape)
```

Comment:
np.zeros?

Reply:
I don't see it

Reply:
Instead of np.array. You rely on it being 0, so you should be explicit about that.